detection-rules/link_credential_phishing_voicemail_language.yml

```
@@ -146,7 +146,7 @@ source: |
           )
           and (
             regex.icontains(.file_name,
-                            '(?:voice|audi[o0]|call|missed|caii|mail|message|recording|call|caii|transcription|v[nm]|audi[o0]|play|listen|unheard|msg)',
+                            '(?:voice|aud[il][o0]|call|missed|caii|mail|message|recording|call|caii|transcr[il]ption|v[nm]|audi[o0]|play|listen|unheard|msg)',
```

add `1` into the mix?
```suggestion
                            '(?:voice|aud[i1l][o0]|call|missed|caii|mail|message|recording|call|caii|transcr[il1]ption|v[nm]|audi[o0]|play|listen|unheard|msg)',
```